scripts/0_setup.sh

```diff
@@ -17,14 +27,43 @@ fi
 
 source .dfl/env/bin/activate
 
+python -m pip install --upgrade pip
+
 version=$(python -V | cut -f 2 -d ' ' | cut -f 1,2 -d .)
 reqs_file='requirements.txt'
 
+version_suffix=''
 if [[ ! -z "$version" && -f "requirements_$version.txt" ]]; then
-  reqs_file="requirements_$version.txt"
+  version_suffix="_$version"
 fi
 
+architecture_suffix=''
+if is_arm64 && [ -f "requirements${version_suffix}_arm64.txt" ]; then
+  architecture_suffix="_arm64"
+fi
+
+reqs_file="requirements${version_suffix}${architecture_suffix}.txt"
+
 echo "Using $reqs_file for $(python -V)"
-pip install -r $reqs_file
+
+if is_arm64; then
+  if [[ -z "$(brew ls --versions hdf5)" ]]; then
+    echo "ERROR: HDF5 needs to be installed to run DeepFaceLab on M1 chip."
```

Comment:
Should we add a note for this in the readme - similar to what we do for ffmpeg?

Reply:
Yes, good point! Will update it once we make it work.